content/pt-br/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Configurando `RunAsUserName` Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como usar a configuração `runAsUserName` para Pods 
+e contêineres que serão executados em nós Windows. Isso é aproximadamente 
+equivalente à configuração `runAsUser` específica do Linux, permitindo a você 
+executar aplicativos em um contêiner com um nome de usuário diferente do padrão. 
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando Kubectl
+deve ser configurada para se comunicar com o seu cluster. Espera-se que o cluster
+tenha nós `worker Windows`, onde os Pods com contêineres executando as cargas de trabalho do Windows,
+serão agendados.
+
+<!-- steps -->
+
+## Defina o nome de usuário para um Pod
+
+Para especificar o nome de usuário com o qual executar os processos de contêiner do Pod, 
+inclua o campo `securityContext` ([SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)) 
+na especificação do Pod, e dentro dela, o campo `WindowsOptions` ([WindowsSecurityContextOptions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)) 
+contendo o campo `runAsUserName`.
+
+As opções de contexto de segurança do Windows que você especificar para um Pod, 
+se aplicam a todos os contêineres do Pod, inclusive os de inicialização.
+
+Aqui está um arquivo de configuração para um Pod do Windows que possui o campo 
+`runAsUserName` definido:
+
+{{< codenew file="windows/run-as-username-pod.yaml" >}}
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/windows/run-as-username-pod.yaml
+```
+
+Verifique se o contêiner do pod está em execução:
+
+```shell
+kubectl get pod run-as-username-pod-demo
+```
+
+Abra um shell para o contêiner em execução:
+
+```shell
+kubectl exec -it run-as-username-pod-demo -- powershell
+```
+
+Verifique se o shell está executando com o nome de usuário correto:
+
+```powershell
+echo $env:USERNAME
+```
+
+A saída deve ser:
+
+```
+ContainerUser
+```
+
+## Defina o nome de usuário para o contêiner
+
+Para especificar o nome de usuário com o qual executar os processos de um contêiner, 
+inclua o campo `SecurityContext` ([SecurityContext] 
+(/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)) 
+no manifesto do contêiner, e dentro dele, o campo `WindowsOptions` 
+([WindowsSecurityContextOptions] (/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)) 
+contendo o campo `runAsUserName`.
```

Comment:
These links are not rendering correctly because of the extra empty space between `[]` and `()`.